tests/run/worktree/data/ansible/playbook.yml

```diff
@@ -1,6 +1,12 @@
-- hosts: all
+- name: copy and check test file
+  hosts: all
   tasks:
-    - copy:
+    - name: copy test file
+      copy:
         src: testfile
         dest: /tmp/testfile
-    - command: grep PASS /tmp/testfile
+        mode: preserve
+    - name: check test file
+      command: grep PASS /tmp/testfile
+      register: output
+      changed_when: output.rc != 0
```

Comment:
maybe also `changed_when: false` ? we do not care about idempotency in these?

Reply:
`changed_when: false` is used instead.